src/main/webapp/WEB-INF/tags/instructor/search/studentsSearchResults.tag

```diff
@@ -7,9 +7,12 @@
 <div class="panel panel-primary">
     <div class="panel-heading">
         <strong><jsp:doBody/></strong>
+        <div class='display-icon pull-right'>
```

Comment:
Similarly, there shouldn't be a need to change this tag as this tag is for search results of students, rather than comments for students.